app/validators/claim/base_claim_validator.rb

```diff
@@ -376,9 +376,26 @@ def increased_travel?
   def validate_earliest_representation_order_date
     return unless @record.case_type&.name == 'Elected cases not proceeded'
     return unless @record.earliest_representation_order_date
+    return if allow_elected_case_not_proceeded?
+
+    if main_hearing_date_enabled?
+      @record.errors.add(:earliest_representation_order_date,
+                         'invalid for elected case not proceeded and main hearing date')
+    else
+      @record.errors.add(:earliest_representation_order_date, 'invalid for elected case not proceeded')
```

Comment:
This is not blocking merging as it pre-dates this PR (and occurs in various other places in the validators) but we could put these messages in translations files. I see that someone had the good intention of having a generic `error_message_for` method in the `BaseValidator` class but it is not actually used anywhere. Maybe something to consider resurrecting?